omod/src/main/webapp/pages/login.gsp

```diff
@@ -4,160 +4,166 @@
     ui.includeCss("pihcore", "login.css")
     ui.includeJavascript("pihcore", "account/login.js")
 %>
-
```

Comment:
You'll want to hide whitespace in the PR UI in order to see the changes in this file, I must have adjusted the indentations.